platform/broadcom/sonic-platform-modules-nexthop/common/etc/systemd/system/swss.service.d/override.conf

```diff
@@ -0,0 +1,3 @@
+[Unit]
+Wants=pmon.service
```

Comment:
why do we need this change?

Reply:
this change means the swss will not start till pmon is started? @judyjoseph for viz...